### DISCUSSION TITLE
Profile Suspended with No explanation?

### DISCUSSION THREAD

**Discussion by sk-vignesh** [community/community#111350] [Other Feature Feedback, Questions, & Ideas]
### Account Suspension without support

Question

### Body

We have been a long-standing paid GitHub customer, utilizing the platform to host and manage nearly a decade's worth of work across various repositories. We were recently notified that our account was suspended but were not provided with a clear explanation or a defined process for appealing and resolving the issue. As paying customers who have invested significant time and resources into GitHub, we expect to receive prompt and transparent communication regarding account issues and a fair process for redressing grievances. The current need for more clarity and responsiveness is concerning. Please review our case urgently and provide a detailed explanation for the suspension and the steps we can take to reinstate our account if there is a misunderstanding. I would say that open communication and due process are essential for maintaining trust in your services.  I am afraid of losing this account due to a random angry administrator who does not want to understand the genuine concerns on our side, and I am open to discussing it.

**Comment by ettaboyle** [comment]
Hi @sk-vignesh, The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :) 

We cannot answer or respond to any account specific questions on this public platform.

**Comment by sk-vignesh** [comment]
I am trying to get attention as a support ticket is totally ignored. Even if our account did something that would make the world break down, it is totally unacceptable to not assign a reason for the account to be suspended and not respond for a week. I think as a policy, this is worth contesting. Regardless of what happens to our account, closing down is another example of not being open to understanding the issue.

**Comment by ghost** [comment]
same issues here, why no one replies!!!!!, is github still active? do they only care paid customers? I even have github copilot, and I could not use it any more. that is crazy support system.!!!!!!
